### ISSUE TITLE
OrderBy/ThenBy clause incorrect with expanded properties

### ISSUE THREAD

**Issue by phatcher** [object/Simple.OData.Client#313]
I'm trying to order by result set by a couple of expanded properties 

```
 var result = await Client.For<RoleMember>()
                 .Expand(x => new { x.Role, x.Party })
                 .OrderBy(x => x.Role.Name)
                 .ThenBy(x => x.Party.Name)
                 .FindEntriesAsync();
```

what gets requested is

```
 GET: http://localhost/odata/RoleMember?$expand=Role,Party,Asset&$orderby=Role/Name,Party/Name&$orderby=Role/Name,Party/Name 
```

which generates a 404 probably due to the duplicated orderby clause. 
If I remove the ThenBy clause I get 

```
GET: http://localhost/odata/RoleMember?$expand=Role,Party&$orderby=Role/Name
```

which returns the data correctly

**Comment by HEBOS** [comment]
This is entirely true. 
Same here also. 
This should be resolved ASAP.

**Comment by haharoi** [comment]
I heavily use Simple.OData.Client in my windows desktop application and faced the same issue. I resolved the issue for my case, you can try using my version with the fix from https://github.com/haharoi/Simple.OData.Client.
Pull request created: https://github.com/object/Simple.OData.Client/pull/354.

**Comment by object** [comment]
I suppose this is fixed in 4.25.0.
